templates/common/navbar.html

```diff
@@ -19,12 +19,7 @@
                     </li>
                     <li class="nav-item">
                         <a class="nav-link" href="{% url 'portal:apps' %}" target="_self" title="Apps">
-                            Apps
-                        </a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="{% url 'models:index' %}" target="_self" title="Models">
-                            Models
+                            Apps and models
```

Comment:
```suggestion
                            Apps and Models
```

Not insisting, just feels better somehow

Reply:
We don't capitalize other main menu items so we will keep it like this for consistency.